readme.md

```diff
@@ -1,30 +1,28 @@
-# CCS specifications
+# Contest Control System specifications
 
-There are multiple versions of the CCS specifications available below. 
+This repository contains a set of related specifications for
+interoperability between different contest control systems and tooling
+that interact with it. These specifications have been designed and
+used in the contest of the ICPC Word Finals, and have also been used
+at various (sub)regionals; they are meant to be useful outside an ICPC
+context as well.
 
-Stable versions will use the naming scheme `<yyyy>-<mm>`. The meaning of these
-will *not* change, but trivial changes such as fixing spelling and grammar, as
-well as adding improved examples or explanations is allowed.
+The following specifications are present:
 
-Draft versions will either be named `<yyyy>-<mm>-draft` if there is a target
-release date, or simply `draft` for the most bleeding edge (and least stable)
-version. These can change in any way at any time.
+* Contest Control System requirements for the ICPC World Finals
+* Contest API: an API specification for accessing information
+  provided by a CCS.
+* Contest Archive Format: a format closely related to the Contest API
+  for storing a contest on disk for archival.
 
-* [draft](/draft)
-* [2021-11-draft](/2021-11-draft)
-* [2020-03](/2020-03) - Latest stable version
+There are multiple versions of the CCS specifications available on the
+[documentation pages](https://ccs-specs.icpc.io/).
+
+This is the draft of some future version of the CCS specification.
 
 ## References
 
-Website: <https://ccs-specs.icpc.io>
-Github: <https://github.com/icpc/ccs-specs>
+* Website: <https://ccs-specs.icpc.io>
+* Github: <https://github.com/icpc/ccs-specs>
 
 The problem package format specification can be found at: <https://icpc.io/problem-package-format/>
-
-## Development notes
```

Comment:
This is also the github readme. I'd prefer to keep this.

Reply:
The issue is that this doesn't work out of the box anymore, since I moved the jekyll config to the common `gh-pages` branch to remove duplication and have a consistent config across all branches.

An option would be to link to the `gh-pages` branch readme and explain there how to build. I didn't test though whether that works out of the box on your local machine.